src/main/kotlin/io/kuzzle/sdk/protocol/Http.kt

```diff
@@ -15,14 +15,18 @@ open class Http : AbstractProtocol {
    * Contrsructor of the Http protocol
    * @param Uri request uri
    */
-  constructor(uri: URI) {
+  constructor(url: String) {
+    if (url.contains("/_query")) {
```

Comment:
Useless check IMHO, if the user use a wrong URI then it's his fault.

They should only use URI without path